### PR TITLE
Update renovate config

### PR DESCRIPTION
**Pull request by frbuceta**
- Every Monday I see that the dependency lock file is not updated, for that I have deactivated the rate limit for it.
- I have also created other groups (see code)

## Checklist

- [x] DCO (Developer Certificate of Origin) [signed in all commits](https://loopback.io/doc/en/contrib/code-contrib.html)
- `npm test` passes on your machine
- New tests added or existing tests modified to cover all changes
- [x] Code conforms with the [style guide](http://loopback.io/doc/en/contrib/style-guide.html)
- API Documentation in code was updated
- Documentation in [/docs/site](../tree/master/docs/site) was updated
- Affected artifact templates in `packages/cli` were updated
- Affected example projects in `examples/*` were updated

👉 [Check out how to submit a PR](https://loopback.io/doc/en/lb4/submitting_a_pr.html) 👈

### CODE REVIEW

File: .github/renovate.json [hunk 1]
```diff
@@ -20,14 +20,16 @@
     "**/__tests__/**",
     "**/test/**"
   ],
+  "labels": ["dependencies"],
```
Comment (bajtos) [inline .github/renovate.json:23]: Nice touch!

**Review by bajtos — APPROVED** [review]

File: .github/renovate.json (hunk 1 above)
Comment (frbuceta) [reply .github/renovate.json:23]: Thanks

**Comment by frbuceta** [comment]
@bajtos thanks for the approval

**Comment by frbuceta** [comment]
I had to remove `[skip ci]` it won't let me merge with it

**Pull request merged**